src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -1335,7 +1340,7 @@ bool SequentialSfMReconstructionEngine::Resection(const size_t viewIndex)
     }
 
     // E. Update the global scene with the new found camera pose, intrinsic (if not defined)
-    if (b_new_intrinsic)
+    if (pinhole_cam == nullptr)
```

Comment:
`// create a new intrinsic id only if the view didn't already have an associated intrinsics` just to make it clear